### PR TITLE
Remove how to KB articles

### PR DESCRIPTION
**Pull request by NimRegev**
Removed unused invalid How to KB articles after porting content to core doc as relevant

### CODE REVIEW

File: _docs/administration/account-user-management/create-codefresh-account.md [hunk 1]
```diff
@@ -167,50 +167,25 @@ max-width="40%"
...
+You want to rename your existing account, either due to a sign-up error or change of business name.
 
+At this time, only Codefresh Support can rename accounts.  
+Please submit a support ticket with the following details:
+* Current account name
+* New account name  
+  If the new account name already exists due to a sign-up error, please indicate this as well.
 
-## Other Git connection options
+##### Delete Codefresh account
```
Comment (hauskaffee) [inline _docs/administration/account-user-management/create-codefresh-account.md:183]: Can we add that only Account Owners can delete the account.

**Review by hauskaffee — CHANGES_REQUESTED** [review]

**Comment by hauskaffee** [comment]
The file `out-of-disk-space.md` might be good to keep unless that information is elsewhere.

File: _docs/administration/account-user-management/create-codefresh-account.md (hunk 1 above)
Comment (NimRegev) [reply _docs/administration/account-user-management/create-codefresh-account.md:183]: Done

**Comment by NimRegev** [comment]
> The file `out-of-disk-space.md` might be good to keep unless that information is elsewhere.

Please confirm that it's covered here: https://codefresh.io/docs/docs/pipelines/pipelines/#set-minimum-disk-space-for-a-pipeline-build

**Comment by hauskaffee** [comment]
@NimRegev Thank you. Forgot about that page and the Ui will contain the information needed.   Thank you.

**Review by hauskaffee — APPROVED** [review]

**Pull request merged**